test/test_gt_pol.py

```diff
@@ -0,0 +1,530 @@
+
+libpath = f"{cwd}/../"
+
+if libpath not in sys.path:
+    sys.path.append(libpath)
+
+import unittest
+from pathlib import Path
+
+# import matplotlib.pyplot as plt
+from nrcatalogtools.maya import MayaCatalog
+from nrcatalogtools.utils import maya_catalog_info
+from pycbc import pnutils
+
+##############################
+# Pycbc
+##############################
+from pycbc.filter.matchedfilter import match
+
+# pycbc
+# from pycbc.waveform import td_approximants
+from pycbc.types.timeseries import TimeSeries
+
+#################################
+# waveformtools
+#################################
+from waveformtools.waveforms import modes_array
+from waveformtools.waveformtools import message, roll, xtract_camp_phase
+
+# unittest helper funcs
+from helper import rms_errs
+
+# from pycbc.waveform.utils import coalign_waveforms
+
+
+# import matplotlib.pyplot as plt
+
+
+######################################
+# Simulation properties
+######################################
+
+sim_name = "GT0001"
+
+# Parameters
+total_mass = 40
+distance = 1000
+# inc = np.pi/3
+# np.pi/3 - 0.0001
+# coa_phase = np.pi/6
+# coa_phase0 = np.pi/6
+delta_t = 1.0 / (4 * 2048)
+
+
+message(f"Simulation {sim_name}", message_verbosity=2)
+
+nrcat_use_lal_conven = True
+wftools_use_lal_conven = True
+lal_use_coa_phase_as_phi_ref = False
+test_wrt_wftools = False
+
+######################################
+# Waveform comparison function
+######################################
+
+
+def GetPolsToCompare(sim_name, total_mass, distance, inclination, coa_phase, delta_t):
+    """Get polarization time-series to compare from
+    nrcatalogtools, waveformtools and lal methods.
+
+    Parameters
+    ----------
+    sim_name : str
+               The simulation name
+    total_mass : float
+                 The total mass in solar mass
+                 units.
+    distance : float
+               The distance in Mpc
+    inclination : float
+                  The inclination angle in radians
+    coa_phase : float
+                The coalescence phase in radians
+    delta_t : float
+              The sampling time step in
+              seconds.
+    Returns
+    -------
+    waveforms : dict
+                The waveform polarizations from nrcatalogtools,
+                waveformtools and lal.
+    """
+
+    #######################
+    # Get GT polarizations
+    #######################
+
+    # sc = sxs.Catalog.load(download=True)
+    # rc = RITCatalog.load(verbosity=5, download=True)
+    mc = MayaCatalog.load(verbosity=1, download=True)
+
+    mwf1 = mc.get(sim_name)
+
+    hpc = mwf1.get_td_waveform(
+        total_mass=total_mass,
+        distance=distance,
+        inclination=inclination,
+        coa_phase=coa_phase,
+        delta_t=delta_t,
+    )
+    hpc_pycbc = hpc  # mwf.to_pycbc(hpc)
+
+    # Minus sign to rotate by pi/2
+    hp_n, hx_n = hpc_pycbc.real(), hpc_pycbc.imag()
+
+    if nrcat_use_lal_conven is True:
+        hp_n, hx_n = hp_n, -hx_n
```

Comment:
Would this be needed now that #36 has been merged?

Reply:
No

Reply:
Excellent, in that case please remove this if-block, and if not needed, remove the flag `nrcat_use_lal_conven` entirely.